### PR TITLE
Cleaned Up Time Table

### PR DESCRIPTION
**Pull request by donhusa**
Removed unnecessary loop

### CODE REVIEW

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

**Comment by jedi22** [comment]
ok to test

**Comment by jedi22** [comment]
ok to test

File: osquery/tables/utility/time.cpp [hunk 1]
```diff
@@ -15,8 +15,6 @@
 namespace osquery {
 namespace tables {
 
-const int kNumCols = 1;
```
Comment (theopolis) [inline osquery/tables/utility/time.cpp]: lol

**Comment by marpaia** [comment]
I lol'd. This is here because, back in the day, when I was testing RocksDB stuff, I needed a table that would have tons of changing results. I would change `kNumCols` to be 50 or what have you and performance monitor the process. Good times.

**Pull request merged**